src/styles/index.scss

```diff
@@ -1,3 +1,68 @@
+html {
+  height: 100%;
+}
```

Comment:
It seems that you're setting the height of the html to 100%. While this is not necessarily wrong, it's uncommon to directly set the height on the html element. It's more common to set it on the body element or on a wrapper div. This is not a critical issue, but it's something to keep in mind.

Reply:
ok thanks